io/eolearn/io/local_io.py

```diff
@@ -2,19 +2,59 @@
 Module containing tasks used for reading and writing to disk
 """
 
-import os.path
-
+import os
 import datetime
+import warnings
+from abc import abstractmethod
+
+import dateutil
 import rasterio
 import numpy as np
-from pyproj import Proj, transform
-from sentinelhub import CRS
-from sentinelhub.time_utils import iso_to_datetime
 
-from eolearn.core import EOTask, SaveToDisk, FeatureType
+from sentinelhub import CRS, BBox
+
+from eolearn.core import EOPatch, EOTask
+
+
+class BaseLocalIo(EOTask):
+    """ Base abstract class for local IO tasks
+    """
+    def __init__(self, feature, folder=None, *, image_dtype=None, no_data_value=0):
+        """
+        :param feature: Feature which will be exported or imported
+        :type feature: (FeatureType, str)
+        :param folder: A directory containing image files or a path of an image file
+        :type folder: str
+        :param image_dtype: Type of data to be saved into tiff image
```

Comment:
saved or loaded*

Reply:
Changing to imported or exported to be more consistent.